packages/scripts/config/puppeteer.config.js

```diff
@@ -1,7 +1,10 @@
+const { launchArgs } = require( 'puppeteer-testing-library' );
+
 module.exports = {
 	launch: {
 		devtools: process.env.PUPPETEER_DEVTOOLS === 'true',
 		headless: process.env.PUPPETEER_HEADLESS !== 'false',
 		slowMo: parseInt( process.env.PUPPETEER_SLOWMO, 10 ) || 0,
+		args: launchArgs(),
```

Comment:
Noting, that it silently enables the testing library for all projects using `@wordpress/scripts`. One way to avoid that for start can be achieved by creating a local Puppeteer config that extends this config and adds this integration.

Reply:
Good point! But I don't think it matters though. This simply just add a flag when launching the chromium instance (`--enable-blink-features=ComputedAccessibilityInfo`), it doesn't affect any tests. We can still create a separate config though, but I'm just not sure it worth the efforts.

Reply:
Can you enable the flag in the existing config instead?

Reply:
I'm not sure what you mean 🤔 ?

Reply:
If this is only one flag in the `jest-puppeteer` config to set why does it need to use a package dependency?

Reply:
Just that it's easier and the users don't have to know about this flag at all. If the library requires more arguments in the future, the users don't have to update it as well. If Firefox/Playwright requires different arguments, we can also handle the logics inside the `launchArgs()` public API.

It's fine to me to just inline the string here though, just curious what's the drawbacks of including a package dependency in `scripts` other than the increasing size 🤔 ?

Reply:
Yes, but we agreed to test it inside Gutenberg first. Besides, we should keep `@wordpress/scripts` as minimal as possible to decrease maintenance work. The goal of this package is to hide all the complexity of setting up the project for WP block development. It should be less opinionated than Gutenberg.